rules/recording_test.go

```diff
@@ -120,6 +120,21 @@ func setUpRuleEvalTest(t require.TestingT) *promql.Test {
 	return suite
 }
 
+func setUpRuleEvalExemplarTest(t require.TestingT) *promql.Test {
+	suite, err := promql.NewTest(t, `
+		load 1m
+			metric{label_a="1",label_b="3"} 1
+			metric{label_a="2",label_b="4"} 10
+
+		load_exemplars 1m
+			metric{label_a="1",label_b="3"} 1
+			metric{label_a="2",label_b="4"} 10
```

Comment:
Can you explain this?  I expect an exemplar to attach to an existing series and add some more labels (in practice, exactly one label "traceid" is typical).

Reply:
Agree too, it's weird. I wonder if we could extend load to add series and some notion of exemplar somewhere?

Reply:
I agree with Bryan, we should work closer to the openmetrics format and use traceid as an exemplar.
`load 1m metric 1 1 1 1 # {traceid="1234"} 2 10 # {traceid="1235"} 4 10 # {traceid="1236"} 6 10`